### DISCUSSION TITLE
Handling of null Double

### DISCUSSION THREAD

**Discussion by adamkorynta** [USACE/cwms-data-api#761] [General Aesthetics]
What is the best way to handle null Double returned by the database in CDA? The JDBC client code substitutes Double.NaN or HEC_UNDEFINED_DOUBLE constant (-Float.MAX_VALUE) for primitive access.

My opinion is returning null as this is what the TimeSeries class does. The location level retrieval, however, uses the HEC_UNDEFINED_DOUBLE instead.

**Comment by MikeNeilson** [comment]
With #913 it would seem we decided that `null` is officially the missing value and should just be used, but the API will except -Float.MAX_VALUE and behave accordingly, setting to null before passing on to the database.
